cmd/synapse/main.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2025 The Synapse Authors
```

Comment:
Can we make sure this is the same one we have in the Synapse repo?

Reply:
+1 fixed in [e1da357](https://github.com/apache/synapse-go/pull/2/commits/e1da3576a03b9d471a9a1310e49b3374bade0d56)